docs/en/notes/changelog.md

```diff
@@ -1,5 +1,39 @@
 # Changelog of v1.1
 
+### v1.3.0 (18/10/2023)
+
+#### Highlights
+
+- Support [CENet](https://arxiv.org/abs/2207.12691) in `projects`. (#2619)
+- Enhance Demos with new 3DInferencer. (#2763)
+
+#### New Features
+
+- Support [CENet](https://arxiv.org/abs/2207.12691) in `projects`. (#2619)
+
+#### Imporvments
+
+- Enhance Demos with new 3DInferencer. (#2763)
+- Add BEV-based detection pipeline in NuScenes Dataset tutorial. (#2672)
+- Add the new config type of Cylinder3D in `mmdet3d/configs`. (#2681)
+- Update New Config Type. (#2655)
```

Comment:
Note the usage of block letters and make it consistent with previous release notes

Reply:
Also modify other sentences